### DISCUSSION TITLE
New SFX additions request discussion

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#20030] [Feature requests]
Hia,
I wanted to ask the community what extra SFX people would like to hear in the game.

Currently in the pipeline
- Night ambiance for the night mode
- LSM/Booster roar
- Flat ride ambience
  - Top spin

**Comment by IntelOrca** [comment]
Wind ambiance when you are zoomed out.

Locomotion has ocean ambience when panning over the ocean.
Locomotion also had wind ambiance when panning over high snowy mountains.

**Reply by karst** [reply]
Will see what I can do here. If theres a windy day I can record that. I will be at the Atlantic Ocean and Mediteranian Sea in August if everything goes to plan so I will definitely record some waves.

**Reply by IntelOrca** [reply]
For this and the waterfall sounds, I assume there are plenty of royalty free sound effects that can be used if you can't record them yourself.

**Reply by karst** [reply]
Yeah most definitely!

**Comment by 733737** [comment]
what is night mode? I thought day night cycle was visual only so how would you tell the game to change crowd noise? Would night mode just be for custom night palettes?

**Reply by karst** [reply]
I'd imagine so. The ambiance is already recorded and a PR is made for it, but there's currently no code that could play said noise

**Comment by IntelOrca** [comment]
Waterfall noise. Could scale up the more there are together and how close you zoom into them.

**Reply by karst** [reply]
I can probably record this when I am on vacation in France  (near the Alpes) in August.

**Reply by spacek531** [reply]
Would this be created based on the waterfall track element or waterfall scenery?

**Reply by IntelOrca** [reply]
The waterfall scenery. The river rapids already have an ambient white noise effect.

**Reply by spacek531** [reply]
Ambient white noise on the vehicles or track?

**Comment by 733737** [comment]
is there currently any scenery object that emits audio?

**Reply by karst** [reply]
I would really like a scenery item that extends the range of ride music. So a scenery item that would have a UI similar to the banner, but with a drop down menu with rides, and you can select which ride to synchronize it too. Would probably be nice to allow rides to synchronize their music with eachother, too.  If you'd click the checkbox at the ride music the list would change from ride music to other ride names. See https://github.com/OpenRCT2/OpenRCT2/discussions/19711

**Reply by ocalhoun6** [reply]
Or just a "loudspeaker" scenery item that can play arbitrary music/sounds. Maybe ride music, maybe something else.

**Comment by spacek531** [comment]
Diesel train horn. Steam whistle exists, but an air horn does not.

**Reply by karst** [reply]
It's actually a great idea to allow some custom rides to pull from sound groups that don't necessarily exist yet. Perhaps the train horns could also sound different depending on what steam loco you select.
